script/zip_manifests/dist_zip.linux.x64.manifest

```diff
@@ -68,5 +68,6 @@ snapshot_blob.bin
 swiftshader/libEGL.so
 swiftshader/libGLESv2.so
 swiftshader/libvk_swiftshader_icd.json
+crashpad_handler
```

Comment:
we shouldn't be shipping this binary on Linux since we don't use it. (we still use breakpad). opened #20783 to track.

Reply:
manifest complains about it being generated, might be coming from a transitive dep. Didn't investigate, can you look into it ?